src/desdemona/ui/table.cljs

```diff
@@ -14,7 +14,19 @@
   ((if (contains? coll k)
      disj conj) coll k))
 
-(defn columns-toggler-component
+(defn make-toggler-affix
+  []
+  (let [toggler (sel1 :#columns-toggler)
```

Comment:
At this point, yeah. We might make it more reusable in the future as needed (but I'm assuming you'd rather have me do that now?)